### DISCUSSION TITLE
Docker install Guide

### DISCUSSION THREAD

**Discussion by afzl-wtu** [Ylianst/MeshCentral#6330] [Issues] [answered]
I could not find anywhere in official repo and install guide how to install with docker. In past I was using docker image:
```
ghcr.io/ylianst/meshcentral:1.1.10@sha256:bda056772d10abf1cfa90d9180d970b244.....
```

But I do not remember from where I got this.

**Comment by si458** [comment]
you dont need to specify the sha256 value, you only need the image name and tag number
`ghcr.io/ylianst/meshcentral` -> defaults to `latest` image/version (1.1.27 at the current time/date)
`ghcr.io/ylianst/meshcentral:1.1.26` -> use certain version (1.1.26 in this case)
`ghcr.io/ylianst/meshcentral:master` -> this is build automatically every time we push a commit to repo for testers

**Comment by dmikoss** [comment, marked as answer]
Docker compose guide:
https://github.com/Ylianst/MeshCentral/tree/master/docker
